backend/src/services/preferences/wishlist.service.ts

```diff
@@ -253,9 +253,13 @@ const getWishlistByUserId = async (
     },
   );
 
+  const wishlistCount =
+    sanitizedModelsWishlist.length + sanitizedComplectationsWishlist.length;
+
   const wishlists: WishlistsResponseDto = {
     models: sanitizedModelsWishlist,
     complectations: sanitizedComplectationsWishlist,
+    count: wishlistCount,
   };
```

Comment:
I would put the counting login on the client side. Then we'll be able to update the liked count right after the like